### ISSUE TITLE
xor, difference are incorrect on large bitmaps

### ISSUE THREAD

**Issue by jhgx** [andreasvc/roaringbitmap#22]
xor:
```
>>> len(RoaringBitmap(range(0, 61440)) ^ RoaringBitmap(range(0, 61440)))
0
>>> len(RoaringBitmap(range(0, 61441)) ^ RoaringBitmap(range(0, 61441)))
65536
```

this also causes issues with `flip_range`.

difference:
```
>>> len(RoaringBitmap(range(0, 61440)) - RoaringBitmap(range(0, 61440)))
0
>>> len(RoaringBitmap(range(0, 61441)) - RoaringBitmap(range(0, 61441)))
61441
```

**Comment by andreasvc** [comment]
Thx!

**Comment by jhgx** [comment]
Thanks for the quick fix!

Not sure if this is part of the same issue but when inverting a bitmap on that same size boundary:

```
>>> len(~RoaringBitmap(range(0, 61440)))
0
>>> len(~RoaringBitmap(range(0, 61441)))
1
```

**Comment by andreasvc** [comment]
It was not the same issue.. fixed in 95ca18f
